algosdk/transaction.py

```diff
@@ -336,46 +335,67 @@ class AssetConfigTxn(Transaction):
             holdings of this asset
         clawback (str, optional): account allowed take units of this asset
             from any account
+        url (bytes, optional): a URL where more information about the asset can be retrieved
+        metadata (bytes, optional): a commitment to some unspecified asset metadata
         note (bytes, optional): arbitrary optional bytes
         gen (str, optional): genesis_id
-        flat_fee (bool): whether the specified fee is a flat fee
+        flat_fee (bool, optional): whether the specified fee is a flat fee
 
     Attributes:
         sender (str)
         fee (int)
         first_valid_round (int)
         last_valid_round (int)
         genesis_hash (str)
-        creator (str)
         index (int)
         total (int)
         default_frozen (bool)
-        unit_name (bytes)
-        asset_name (bytes)
+        unit_name (byte[8])
+        asset_name (byte[32])
         manager (str)
         reserve (str)
         freeze (str)
         clawback (str)
+        url (bytes[32])
+        metadata (byte[32])
         note (bytes)
         genesis_id (str)
         type (str)
     """
 
-    def __init__(self, sender, fee, first, last, gh, creator=None, index=None,
+    def __init__(self, sender, fee, first, last, gh, index=None,
                  total=None, default_frozen=None, unit_name=None,
-                 asset_name=None, manager=None, reserve=None, freeze=None,
-                 clawback=None, note=None, gen=None, flat_fee=False):
+                 asset_name=None, manager=None, reserve=None, 
+                 freeze=None, clawback=None, url=None, metadata=None,
+                 note=None, gen=None, flat_fee=False):
         Transaction.__init__(self,  sender, fee, first, last, note, gen, gh)
-        self.creator = creator
         self.index = index
         self.total = total
         self.default_frozen = default_frozen
         self.unit_name = unit_name
         self.asset_name = asset_name
+        if unit_name != None:
+            if len(unit_name) > constants.unit_name_length:
+                raise error.WrongUnitNameLengthError
+            self.unit_name = unit_name + (constants.unit_name_length - len(unit_name))*b'\x00'
```

Comment:
Note that we're changing the go code to use var length for unitName, AssetURL and assetName